Firmware/Marlin_main.cpp

```diff
@@ -5857,12 +5857,22 @@ SERIAL_PROTOCOLPGM("\n\n");
     Restart the printer-host enable keepalive timer. While the timer has not expired, the printer will enable host specific features.
```

Comment:
Update ### M79 - TODO
`### M79 - Start host timer <a href="https://reprap.org/wiki/G-code#M79:_Start_host_timer">M79: Start host timer</a>`

`Restart the printer-host enable keepalive timer.` -> `Start the printer-host enable keep-alive timer.`

Reply:
Done. See [890bfab](https://github.com/prusa3d/Prusa-Firmware/pull/4479/commits/890bfab946b07fec8b0d391fd8058d40bd68a9c2)